### ISSUE TITLE
"We" means "we the science case investigators" unless otherwise stated

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTScienceCollaborations/ObservingStrategy#489]
This is another latex hop-through, albeit involving several different latex files per chapter... Best undertaken by the chapter editors in parallel I think. Please report in once you have done this check for your chapter! Thanks :-)

* [ ] Introduction @drphilmarshall

* [ ] Example Observing Strategies @ivezic @yoachim

* [ ] Solar System @davidtrilling @rhiannonlynne

* [ ] The Galaxy @akvivas @willclarkson

* [ ] Variable Objects @AshishMahabal @lmwalkowicz

* [ ] Eruptive and Explosive Transients @ebellm @fedhere

* [ ] AGN @ohadshemmer @tanguita

* [ ] Cosmology @MichelleLochner @egawiser

* [ ] Special Surveys @drphilmarshall

* [ ] Synergy with WFIRST @jasondrhodes

* [ ] Trade-offs @StephenRidgway

**Comment by jasondrhodes** [comment]
@drphilmarshall I have done this for WFIRST, but I am not sure I was looking for the right thing.  Can you give a bit more context for this issue?

**Comment by lmwalkowicz** [comment]
Yeah @drphilmarshall I don't actually know what you mean by a LaTeX hop through?

**Comment by drphilmarshall** [comment]
Ah, sorry! More explanation:

In the introduction to the white paper we will make the following statement:

> "We" means "we the science case investigators" unless otherwise stated 

That means that all of the rest of the paper must be consistent with that statement. So, in each chapter, we need to go through every line of latex searching for the string " we " (using our editor's case insensitive search facility) and make sure that all sentences containing the word we comply with the standard we have set ourselves. If we ever use "we" to refer to a group of people other than the current science case's investigators, we will need to explain that, as in "In this case, 'we' refers to the LSST science community, rather than just the present science case's investigators."

I hope this makes sense! If you hit an odd case, please make a note of it in the tex so you can find it again, and post it to this thread. Thank you all!

**Comment by MichelleLochner** [comment]
The "we"s in Cosmology look ok to me.
